playbooks/cdi.yml

```diff
@@ -0,0 +1,11 @@
+---
+- hosts: localhost
+  vars:
+    storage_role: cdi
```

Comment:
storage_role is only for storage flavors (currently storage-cns and storage-demo).  The cdi role will be included in the storage.yml playbook regardless of the flavor used.  You can remove this variable.